### PR TITLE
Sp/concurrent start stop

### PR DESCRIPTION
**Pull request by spickartz**
This patch performs the start/stop of the VMs using a single message per host.

### CODE REVIEW

File: include/poncos/vm_controller.hpp [hunk 1]
```diff
@@ -99,7 +104,7 @@ class vm_controller : public controllerT {
 	// stores the VMs used in the two slots
 	// TODO was originalle an unordered map but the key was never used for a lookup
```
Comment (jbreitbart) [inline include/poncos/vm_controller.hpp]: @spickartz is this todo fixed with the change?

File: include/poncos/vm_controller.hpp [hunk 2]
```diff
@@ -23,6 +23,10 @@
 #include "poncos/poncos.hpp"
 
 #include <fast-lib/mqtt_communicator.hpp>
+#include <fast-lib/message/migfra/pci_id.hpp>
```
Comment (jbreitbart) [inline include/poncos/vm_controller.hpp]: Do we all three includes just for fast::msg::migfra::Start?

**Review by jbreitbart — APPROVED** [review]

File: include/poncos/vm_controller.hpp (hunk 2 above)
Comment (spickartz) [reply include/poncos/vm_controller.hpp]: No, we need both, the `task` and the `result`. However, the following is obsolete and I'll remove this:
```
 #include <fast-lib/message/migfra/pci_id.hpp>
```

File: include/poncos/vm_controller.hpp (hunk 1 above)
Comment (spickartz) [reply include/poncos/vm_controller.hpp]: Yes, I will remove the comment.

**Pull request merged**